api/v1beta1/common_types.go

```diff
@@ -101,6 +101,83 @@ type BarbicanComponentTemplate struct {
 	NetworkAttachments []string `json:"networkAttachments,omitempty"`
 }
 
+// BarbicanLunaTemplate - Variables specific to Thales' Luna HSMs
+type BarbicanLunaTemplate struct {
```

Comment:
This structure should be added the relevant specs - ie. the BarbicanApiSpec and the BarbicanWorkerSpec

Reply:
I'll create another struct called `Pkcs11Template` that will encompass this one. It will then be added to `BarbicanApiSpec` and `BarbicanWorkerSpec`.

Agreed, or do you have another suggestion?